games/openrct2/Portfile

```diff
@@ -18,8 +18,24 @@ long_description    ${description} A construction and management simulation \
                     video game that simulates amusement park management. Requires \
                     files from the original RollerCoaster Tycoon 2 in order to work.
 
-checksums           rmd160  16fe40e2a07034f8571b4020634b23f10506f39a \
-                    sha256  51dd38d028bb352904162f63d1a4093d6580a702d8a4fc54eb98c3466b05ac69
+master_sites        ${github.homepage}/tarball/${git.branch}:main \
+                    https://github.com/${github.author}/objects/releases/download/v1.0.18:objects \
+                    https://github.com/${github.author}/title-sequences/releases/download/v0.1.2c:ts
```

Comment:
there is a slightly different way that we use to add more distfiles to a github port that requires a bit less surgery.

See <https://github.com/macports/macports-ports/blob/225b5074603ea92f63594b4af44efa106e788f1f/games/minetest/Portfile#L21> for another port where I did this.

Reply:
Is my latest push better?

Also I made it so the extraction process is done by MacPorts implicitly and then my Portfile just re-arranges the extracted data.